package-lock.json

```diff
@@ -0,0 +1,2166 @@
+{
```

Comment:
Shouldn't we have a `package.json` file that generated this package-lock?

Reply:
good question :thinking: i'm not sure if we need it, because this is just for serverless and its plugins. The project metadata is already in into `pyproject.toml`.

Reply:
If we don't have a `package.json`, I don't think it makes sense to have a `package-lock.json` either.
I cannot do a `npm install` in the repo with just a `package-lock.json`, so it's useless.

Having said that, I think we actually should commit both of them in the repo, because that's the way to lock the serverless plugins' versions. Otherwise we may get different versions for the plugins every time the project is setup.

This is important especially if some of the serverless plugins we are using affect the final bundle generated by serverless.